Scoop/Controller/TripExtraDetailsViewController.swift

```diff
@@ -0,0 +1,182 @@
+        
+        dateField.placeholder = "mm/dd/yyyy"
+        dateField.textColor = .black
+        view.addSubview(dateField)
+        
+        
+        timeLabel.text = "Time of Trip"
+        timeLabel.textColor = .black
+        view.addSubview(timeLabel)
+        
+        let clockView = UIImageView(image: UIImage(systemName: "clock"))
+        clockView.tintColor = .black
+        view.addSubview(clockView)
+        
+        timeField.placeholder = "1:33 AM"
+        timeField.textColor = .black
+        view.addSubview(timeField)
+        
+        
+        detailsLabel.text = "Other Details"
+        detailsLabel.textColor = .black
+        view.addSubview(detailsLabel)
+        
+        let writeView = UIImageView(image: UIImage(systemName: "square.and.pencil"))
+        writeView.tintColor = .black
+        view.addSubview(writeView)
+        
+        detailsField.placeholder = "enter details..."
+        detailsField.textColor = .black
+        view.addSubview(detailsField)
+
+//        let transImage = UIImage(systemName: "location")
+//        let transView = UIImageView(image: transImage)
+//        transView.tintColor = .black
+//        transView.frame = CGRect(x: 0, y: 0, width: 20, height: 20)
+//        view.addSubview(transView)
+        
+        
+        travelersLabel.snp.makeConstraints { make in
+            make.top.equalToSuperview().offset(300)
+            make.leading.equalToSuperview().offset(80)
+        }
+        
+        peopleView.snp.makeConstraints { make in
+            make.top.equalTo(travelersLabel.snp.bottom).offset(10)
+            make.leading.equalTo(travelersLabel.snp.leading)
+        }
+        
+        count1Field.snp.makeConstraints { make in
+            make.top.equalTo(peopleView.snp.top)
+            make.leading.equalTo(peopleView.snp.trailing).offset(10)
+        }
+        toLabel.snp.makeConstraints { make in
+            make.top.equalTo(peopleView.snp.top)
+            make.leading.equalTo(count1Field.snp.trailing).offset(10)
+        }
+        count2Field.snp.makeConstraints { make in
+            make.top.equalTo(peopleView.snp.top)
+            make.leading.equalTo(toLabel.snp.trailing).offset(10)
+        }
+        
+        dateLabel.snp.makeConstraints { make in
+            make.leading.equalTo(travelersLabel.snp.leading)
+            make.top.equalTo(peopleView.snp.bottom).offset(25)
+        }
+        
+        calendarView.snp.makeConstraints { make in
+            make.leading.equalTo(dateLabel)
+            make.top.equalTo(dateLabel.snp.bottom).offset(10)
+        }
+        
+        dateField.snp.makeConstraints { make in
+            make.leading.equalTo(calendarView.snp.trailing).offset(10)
+            make.top.equalTo(calendarView.snp.top)
+        }
+        
+       timeLabel.snp.makeConstraints { make in
+            make.leading.equalTo(travelersLabel.snp.leading)
+            make.top.equalTo(calendarView.snp.bottom).offset(25)
```

Comment:
trailing anchor here too

Reply:
In general, you should always the leading and trailing anchor for views that take up the whole screen's width